lib/macro.ts

```diff
@@ -0,0 +1,142 @@
+import * as path from "path";
+import { MacroHandler, MacroError, createMacro } from "babel-plugin-macros";
+import { Compiler, ICompilerOptions } from "./index";
+
+const macroHandler: MacroHandler = (params) => {
```

Comment:
Done: https://github.com/gristlabs/ts-interface-builder/pull/24/files?file-filters%5B%5D=.json&file-filters%5B%5D=.ts#diff-e7af3dac85789eec60b8c6dc86d31ef0R15-R23

Reply:
I guess that information (how to use getTypeSuite and getCheckers, with a brief example) would already be in the README documentation, once that step is completed.. do we need to repeat this in inline docs?

This thread lead me to realize we're missing inline docs (i.e. typedoc comments) for getTypeSuite and getCheckers in `macro.d.ts`, so I added that there, and added another comment here, with commit 113aa5e0347dbcc40f44deb5e40acfdbfa8a872f.

Anyways if you want the inline docs (not just the README docs) to have usage examples for getTypeSuite and getCheckers, that's fine, just let me know and I'll include them in the inline docs for those functions. I think it makes more sense to include them in the inline docs for those functions, rather than the inline docs for `macroHandler` which the macro user wouldn't [normally] be calling directly.